DevelopGitHub/GitHubExtension/GitHubExtension.EntryPoint/FrontEnd/src/client/app/layout/ht-top-nav.html

```diff
@@ -1,16 +1,28 @@
-<div class="navbar navbar-inverse navbar-fixed-top" role="navigation" data-ng-controller="topNavController">
-    <div class="container">
-        <div class="navbar-header">
-            <button class="btn btn-success navbar-toggle" data-ng-click="navbarExpanded = !navbarExpanded">
-            </button>
-            <a class="navbar-brand">GitHubExtension</a>
-        </div>
-        <div class="collapse navbar-collapse" data-collapse="!navbarExpanded">
+<nav class="navbar navbar-fixed-top navbar-inverse">
+<div class="container">
+    <div class="navbar-header">
+        <a href="/" class="navbar-brand"><span class="brand-title">GitHubExtension</span></a>
+        <a class="btn navbar-btn navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse">
+            <span class="icon-bar"></span>
+            <span class="icon-bar"></span>
+            <span class="icon-bar"></span>
+        </a>
+    </div>
+    <div class="navbar-collapse collapse">
+        <div class="pull-right navbar-logo">
+            <ul class="nav navbar-nav pull-right">
+                <li ng-repeat="r in vm.navRoutes">
+                    <a ui-sref="{{r.name}}"
+                       ng-bind-html="r.settings.content">
+                    </a>
+                </li>
+            </ul>
             <ul class="nav navbar-nav navbar-right">
-                <li data-ng-hide="!isAuth"><a href="#">Welcome {{userName}}</a></li>
-                <li data-ng-hide="!isAuth"><a href="" data-ng-click="logOut()">Logout</a></li>
-                <li data-ng-hide="isAuth"> <a href="/login">Login</a></li>
+                <li data-ng-hide="!vm.user.isAuthenticated()"><a>Welcome {{vm.user.userName()}}</a></li>
```

Comment:
maybe ng-show will be a bit easy to understand.

I'd change lines 21 and 23 to `data-ng-show`